launchConfig.zon

```diff
@@ -0,0 +1,5 @@
+.{
+    .cubyzDir = "",
+    .headlessServer = false,
+    .autoEnterWorld = "",
+}
```

Comment:
Please don't commit files that are in the .gitignore, this makes it very annoying to change this locally.

Reply:
But you just asked me to change it?

> Could you also adjust the default launchConfig to show all these new fields please?

Reply:
It doesn't seem to create it on launch for me, either on 0.0.0 or latest.

Can you point out where it's created?

Reply:
Got it!